setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 
 
-version = '0.2'
+version = '0.2.1'
```

Comment:
Next time, I would rather bump the version in a dedicated commit (not pull request), I guess.